toolkit/tools/internal/shell/shell.go

```diff
@@ -189,6 +189,25 @@ func ExecuteLiveWithCallback(onStdout, onStderr func(...interface{}), printOutpu
 	return
 }
 
+// ExecuteAndLogToFile runs a command in the shell and redirects stdout to the give file
+func ExecuteAndLogToFile(filepath string, command string, args ...string) {
+	cmd := exec.Command(command, args...)
+	outfile, err := os.Create(filepath)
+	if err != nil {
+		logger.Log.Errorf("Unable to create: %v", err)
+		return
+	}
+	defer outfile.Close()
+	cmd.Stdout = outfile
+	err = cmd.Start()
+	if err != nil {
+		logger.Log.Errorf("Unable to start: %v", err)
```

Comment:
I'd add a bit more details to this log to make it clear what was happening:

```suggestion
		logger.Log.Errorf("Unable to start command '%s %s'. Error: %v", command, strings.Join(args, " "), err)
```

Reply:
sure, will update